<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainSpectrum</class>
 <widget class="QWidget" name="MainSpectrum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="lnbLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #1d1d1f;
color: #b7e869;
font-size: 12px;
font-family: Monospace;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string> LNB </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="loLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #1d1d1f;
color: #b7e869;
font-size: 12px;
font-family: Monospace;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string> DEMOD </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="LCD" name="fcLcd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <longLong>88888888888</longLong>
         </property>
         <property name="thickness">
          <double>0.320000000000000</double>
         </property>
         <property name="zoom">
          <double>0.500000000000000</double>
         </property>
         <property name="backgroundColor">
          <color>
           <red>29</red>
           <green>29</green>
           <blue>31</blue>
          </color>
         </property>
         <property name="foregroundColor">
          <color>
           <red>183</red>
           <green>232</green>
           <blue>105</blue>
          </color>
         </property>
         <property name="min">
          <longLong>0</longLong>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="throttlingLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #1d1d1f;
color: #b7e869;
font-size: 12px;
font-family: Monospace;
font-weight: bold;
</string>
         </property>
         <property name="text">
          <string>  Throttling: OFF</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="Waterfall" name="mainSpectrum"/>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="captureModeLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #1d1d1f;
color: #b7e869;
font-size: 12px;
font-family: Monospace;
font-weight: bold;
</string>
         </property>
         <property name="text">
          <string>  Capture mode: N/A</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="LCD" name="lnbLcd">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <longLong>0</longLong>
         </property>
         <property name="thickness">
          <double>0.320000000000000</double>
         </property>
         <property name="zoom">
          <double>0.500000000000000</double>
         </property>
         <property name="backgroundColor">
          <color>
           <red>29</red>
           <green>29</green>
           <blue>31</blue>
          </color>
         </property>
         <property name="foregroundColor">
          <color>
           <red>183</red>
           <green>232</green>
           <blue>105</blue>
          </color>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="LCD" name="loLcd">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>0</height>
          </size>
         </property>
         <property name="value">
          <longLong>0</longLong>
         </property>
         <property name="thickness">
          <double>0.320000000000000</double>
         </property>
         <property name="zoom">
          <double>0.500000000000000</double>
         </property>
         <property name="backgroundColor">
          <color>
           <red>29</red>
           <green>29</green>
           <blue>31</blue>
          </color>
         </property>
         <property name="foregroundColor">
          <color>
           <red>183</red>
           <green>232</green>
           <blue>105</blue>
          </color>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="MultiToolBox" name="multiToolBox"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LCD</class>
   <extends>QFrame</extends>
   <header>LCD.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Waterfall</class>
   <extends>QFrame</extends>
   <header>Waterfall.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MultiToolBox</class>
   <extends>QWidget</extends>
   <header>MultiToolBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
